### PR TITLE
google: update CONTRIBUTE section, added template modularity and new resources

### PR DESCRIPTION
**Pull request by marcoldp**
#273 #188

### CODE REVIEW

File: google/reader_irregular_cases.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		resources := make([]sqladmin.Database, 0)
+
+		elemList, err := service.List(r.project, elem).Context(ctx).Do()
+		if err != nil {
+			//if 404 remove because resource instance is stop - otherwise fails
+			if strings.Contains(err.Error(), "Invalid request since instance is not running.") {
+				continue
+			}
+			//if resource instance is on-premise - otherwise fails
+			if strings.Contains(err.Error(), "400") {
```
Comment (xlr-8) [inline google/reader_irregular_cases.go:27]: If the instance is on-premise it returns a 400?

**Review by xlr-8 — COMMENTED** [review]
Minor question, rest looks good to me!

File: google/reader_irregular_cases.go (hunk 1 above)
Comment (marcoldp) [reply google/reader_irregular_cases.go:27]: yes i saw this behaviour while testing the infra of OBS

**Review by xescugc — APPROVED** [review]
w8 for @xlr-8

**Pull request merged**